packages/flutter/lib/src/material/bottom_sheet_theme.dart

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+/// Defines default property values for [BottomSheet] widgets.
```

Comment:
At the moment, this theme appears to be for configuring the bottom sheet's material.

Reply:
Modified the phrasing